### DISCUSSION TITLE
Sound on print Complete

### DISCUSSION THREAD

**Discussion by evertonramires** [OctoFarm/OctoFarm#547] [Feature Request]
Would be a great feature for the webapp/gui to play a sound when any of my printers finish a job. Believe could be easy to implement and will reduce printer idle time a lot.
Thanks in advance and keep the great job!

**Comment by davidzwa** [comment]
Hmm I do see your point, but it can also become highly annoying if done too much. Also we have to do a priority trade-off as we have a lot of other stuff requiring our attention at this moment.

May we know how many printers you currently manage and how spread out they are (distance wise)? This can help us picture your scenario better. In my case I can see all the printers within 1 look around and one should always consider that 3D printers are just like a self-driving car: it needs care and attention. So yeah, just help us understand your setup a bit better.

Fyi this plugin was created by a well-known OctoPrint moderator:
https://community.octoprint.org/t/plugin-to-sound-when-print-complete/22662

Also consider push notifications. Wouldnt that be much more friendly?
